privacyidea/lib/resolvers/HTTPResolver.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
+#
+#  Mar, 12 2020 Bruno Cascio
+#  http://www.privacyidea.org
+#
+#  product:  PrivacyIDEA
+#  module:   httpresolver
+#  tool:     HTTPResolver
+#  edition:  Comunity Edition
+#
+#  License:  AGPLv3
+#  contact:  http://www.linotp.org
+#            http://www.lsexperts.de
+#            linotp@lsexperts.de
+#
+# This code is free software; you can redistribute it and/or
+# modify it under the terms of the GNU AFFERO GENERAL PUBLIC LICENSE
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or any later version.
+#
+# This code is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU AFFERO GENERAL PUBLIC LICENSE for more details.
+#
+# You should have received a copy of the GNU Affero General Public
+# License along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+from .UserIdResolver import UserIdResolver
+import requests
+import logging
+import json
+from six.moves.urllib.parse import urlencode
+from pydash import get
+from privacyidea.api.lib.utils import getParam
+
+ENCODING = "utf-8"
+
+__name__ = "HTTP_RESOLVER"
+
+log = logging.getLogger(__name__)
+
+
+class HTTPResolver(UserIdResolver):
+
+    fields = {
+        "endpoint": 1,
+        "method": 1,
+        "requestMapping": 1,
+        "headers": 1,
+        "responseMapping": 1,
+        "hasSpecialErrorHandler": 0,
+        "errorResponse": 0
+    }
+
+    def __init__(self):
+        super().__init__()
```

Comment:
Could You use `super(HTTPResolver, self).__init__()` here to get this working with Python 2.7?